stdlib/FileWatching/src/pidfile.jl

```diff
@@ -307,7 +307,7 @@ function Base.close(lock::LockMonitor)
     if pathstat !== nothing && samefile(stat(lock.fd), pathstat)
         # try not to delete someone else's lock
         try
```

Comment:
This `try/catch` needs to be removed, since `tryrmopenfile` already has its own. I suggest changing that function to returning true-or-exception value so that it is a drop-in replacement for setting `removed` here.

Reply:
Done:

1. `tryrmopenfile` returns `true` or `ex` (instead of `true` or `false`).
2. The `try/catch` on line 309 has been removed.